lib/entity.js

```diff
@@ -14,6 +14,19 @@
 
 'use strict';
 
+// Destroying an entity for which there is i/o activity can
+// results in a SEGFAULT. To mitigate this situation we hold a
+// reference to released handles to prevent them from being
+// GC'ed prematurely.
+const MAX_OBSOLETE_HANDLES_SIZE = 100;
+const OBSOLETE_HANDLES = new Set();
+function registerObsoleteHandle(handle) {
+  OBSOLETE_HANDLES.add(handle);
```

Comment:
I think we can check if the handle has been deleted when calling 'execute`, and remove if the handle has been released.

https://github.com/RobotWebTools/rclnodejs/blob/0ea952f29e6251e229cf132790d61946ca4fca7f/lib/node.js#L150-L155

like
```js
execute(handles) {
  handles.forEach(handle => {
     if (OBSOLETE_HANDLES.indexOf(handle) !== -1) {
       // Remove from OBSOLETE_HANDLES.
      }
  });
...
```
I haven't tested, just in my mind:) thus we don't have to keep these handles for a long time.